src/sentry/relocation/tasks/process.py

```diff
@@ -535,6 +560,12 @@ def uploading_complete(uuid: UUID) -> None:
     retry_backoff_jitter=True,
     soft_time_limit=MEDIUM_TIME_LIMIT,
     silo_mode=SiloMode.REGION,
+    taskworker_config=TaskworkerConfig(
+        namespace=relocation_tasks,
```

Comment:
Should we have a processing_deadline of MEDIUM_TIME_LIMIT to align with the celery configuration?